src/test/java/org/folio/test/support/ApiTests.java

```diff
@@ -166,20 +165,21 @@ protected void removeAllFromTable(String tableName) {
     Criterion criterion = new Criterion();
     if (FEEFINES_TABLE.equals(tableName)) {
       for (AutomaticFeeFineType type : AutomaticFeeFineType.values()) {
-        criterion.addCriterion(createAutomaticFeeFineExclusionCriteria(type), "AND");
+        criterion.addCriterion(createFeeFineExclusionCriteria(type.getId()), "AND");
       }
+      criterion.addCriterion(createFeeFineExclusionCriteria(LOST_FEE_FOR_ACTUAL_COST_ID), "AND");
```

Comment:
Considering that "Lost item fee (actual cost)" is an automatic fee-fine as well:
```
{
 "id": "73785370-d3bd-4d92-942d-ae2268e02ded",
 "feeFineType": "Lost item fee (actual cost)",
 "automatic": true
}
```
it probably makes sense to add it to the AutomaticFeeFineType enum. This way it'll be processed together with others rather than with a separate criterion.